.github/workflows/test-linux.yml

```diff
@@ -15,9 +15,24 @@ jobs:
         uses: actions/setup-python@v1
         with:
           python-version: ${{ env.PYTHON_VERSION }}
+      - uses: actions/cache@v1
+        id: cache
+        with:
+          path: ~/.ccache
+          key: ${{ runner.os }}-ccache
+      - name: Setup ccache
+        run: |
+          sudo apt-get install ccache
+          sudo /usr/sbin/update-ccache-symlinks
       - name: Environment Information
-        run: npx envinfo
+        run: |
+          export CCACHE_DIR=~/.ccache
+          export PATH="/usr/lib/ccache:$PATH"
+          npx envinfo
       - name: Build
-        run: make build-ci -j2 V=1
+        run: |
+          export CCACHE_DIR=~/.ccache
+          export PATH="/usr/lib/ccache:$PATH"
+          make build-ci -j2 V=1
       - name: Test
         run: make run-ci -j2 V=1
```

Comment:
Do we want to enable ccache for building the addons (happens as part of run-ci dependencies) here?

Reply:
done! This is a good excuse to kick off another build and see if the cache improves anything

Reply:
> 
> 
> done! This is a good excuse to kick off another build and see if the cache improves anything

Doesn't look like https://github.com/nodejs/node/pull/32633/checks?check_run_id=558360418 is any faster 😞. It does appear to have picked up the cache from the previous run for the preceding commit.